### DISCUSSION TITLE
상단 탑바를 고정시킬수 있을까요?

### DISCUSSION THREAD

**Discussion by taehyunkim3** [pronist/hELLO#213] [Q&A] [answered]
스킨 너무 잘 쓰고있습니다.

메뉴버튼이 스크롤을 하지 않으면 안보여서, 

처음 들어와도 바로 보이게 하고싶어서요. 

사이드바는 그대로 두고, 위에 탑바?만 고정하고 싶습니다.

html css에 사이드바 부분은 보이는데 탑바는 어디서 수정이 가능할까요.?

여기 discussion에도 사이드바 관련 내용만 있는것 같아서요 ㅠㅠ

좋은 스킨 만들어주셔서 감사드립니다!

**Comment by pronist** [comment]
@taehyunkim3 안녕하세요. 답변이 늦었네요. 스킨의 버전이 어떻게 되십니까? `index.xml` 에서 확인하실 수 있습니다. 과거에 그 부분을 패치를 한 기억이 나는데, 현재 스킨의 버전은 `4.1.3` 버전입니다. 만약 그보다 낮다면 스킨을 다시 다운 받으셔서 업그레이드를 해보시는 것이 좋을 것 같습니다.

**Reply by taehyunkim3** [reply]
오 안녕하세요!! 답변을 지금 확인했네요! 버전은 4.1.3입니다.! 제 블로그는 https://kthdev.tistory.com/ 입니다. 혹시 따로 설정에서 변경이 가능할까요?

**Reply by pronist** [reply, marked as answer]
@taehyunkim3 아아 제가 글을 잘못 이해했던 것 같네요. 탑바를 강제로 고정하는 기능은 현재 스킨에서 제공하고 있지는 않습니다. 탑바의 On/Off 관련 해서는 원본 코드에서는 `Top.pug` 에서 찾아보실 수 있고, HTML/CSS 상으로는 `x-data='top'` 속성이 쓰인 부분을 수정하시면 되겠습니다. 스크롤 이벤트는  ` @scroll.window='toggle'` 에서 등록하고 있으며 ` x-show='open'` 를 통해 On/Off 를 조절합니다.

https://github.com/pronist/tistory-skin-hELLO/blob/9fa166f2cf8164c9302648c5770146027d3b669c/views/Top.pug#L1-L4

탑바를 강제로 고정하려면  `@scroll.window='toggle'`  를 제거하고 `x-show='true'` 와 같이 처리해보시기 바랍니다.

**Reply by taehyunkim3** [reply]
너무 늦게 확인했네요ㅠㅠ 감사합니다!!! 한번 해볼께요!!
